apps/parsley/src/test_utils/index.tsx

```diff
@@ -95,6 +95,41 @@ const renderWithRouterMatch = (
   };
 };
 
+/**
+ * `overwriteFakeTimers` is a workaround for a bug in @testing-libray/react.
+ * It prevents Vitest's fake timers from functioning with user-event.
+ * https://github.com/testing-library/react-testing-library/issues/1197
+ * @returns an anonymous function overwriting jsdon's Jest field
+ */
+const overwriteFakeTimers = () => {
+  const globalJest = globalThis.jest;
+
+  globalThis.jest = {
+    ...globalThis.jest,
+    advanceTimersByTime: vi.advanceTimersByTime.bind(vi),
+  };
+
+  return () => {
+    globalThis.jest = globalJest;
+  };
+};
+
+/**
+ * `stubGetClientRects` fixes a fallbackFocus error introduced by focus-trap.
+ * focus-trap only offers legacy CommonJS exports so it can't be mocked by Vitest.
+ * Instead, spoof focus-trap into thinking there is a node attached.
+ * https://stackoverflow.com/a/75527964
+ */
+const stubGetClientRects = () => {
+  const { getClientRects } = HTMLElement.prototype;
+  HTMLElement.prototype.getClientRects = function () {
```

Comment:
focus-trap 😾 Could we can disable the eslint rule here? (unless we plan to change it)